vendor/oras.land/oras-go/v2/internal/docker/mediatype.go

```diff
@@ -1,13 +1,22 @@
-Copyright 2014 gRPC authors.
-
+/*
+Copyright The ORAS Authors.
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+http://www.apache.org/licenses/LICENSE-2.0
 
 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
+*/
+
+package docker
+
+// docker media types
+var (
```

Comment:
Wonder if it should consgt or var?

Reply:
This is the vendor code right?

Reply:
i see

Reply:
@binbin-li Good catch. Addressed in https://github.com/oras-project/oras-go/pull/152